update readme with orders (#4)
Enrise/paynl-nodejs-sdk: README.md

## Usage

- Import 'paynl-sdk' in your file.
- [Register](https://signup.pay.nl/welcome) for a PAY.nl account at.
- In the PAY.nl admin panel, navigate to Manage -> Services and click the SL-code on the left.
- From the popup use the API token and service location ID, and configure them in the SDK.

## Examples

> Some of the basic examples are listed here, for the full list of examples, please take a look at the examples directory [here](https://github.com/paynl/nodejs-sdk/tree/master/src/examples)

Reviewer: ```suggestion
Some of the basic examples are listed here, for the full list of examples, please take a look at the examples directory [here](https://github.com/paynl/nodejs-sdk/tree/master/src/examples).
```

> All examples start with requiring paynl-sdk and setting the API token and service location ID.

Reviewer: ```suggestion
All examples start with importing the `paynl-sdk` and creating a client using the API token and service location ID.
```

```typescript
import { createPayNLClient } from 'paynl-sdk';

const payNL = createPayNLClient({ apiToken: '****************************************', serviceId: 'SL-####-####' });
```

## Orders

Orders use the new Transaction Gateway Unit API. View the [examples](https://github.com/paynl/nodejs-sdk/tree/master/src/examples/connect) or [online documentation](https://developer.pay.nl/reference/api_create_order-1) for a complete overview of what is possible.

### Creating an order

The most minimal request to create an order includes the amount:

```typescript
const order = await payNL.Orders.create({
    amount: {
        value: 1000, // in cents
        currency: 'EUR',
    },
});
```

For testing purposes you may enable the sandbox mode:

```typescript
const order = await payNL.Orders.create({
    amount: {
        value: 1000, // in cents
        currency: 'EUR',
    },
    integration: {
       test: true,
    },
});

> ```

Reviewer: Should we add more information about what to do with the order object. For example explain where to redirect the user to?

Author: I'd say that is more for the examples or docs than the readme, which gives 'a taste' of what it can do.
But feel free to add such texts if you think otherwise.

### Fetching an order

After an order is created you should store its ID, which you need to fetch or change it later on.
To fetch an order or just its status:

```typescript
const orderId = '########-####-####-####-############';
const order = await payNL.Orders.get(orderId);
const orderStatus = await payNL.Orders.status(orderId);
```

## Basic transactions

> [!IMPORTANT]
> Orders created with the transaction API will not be available in the new transaction gateway unit.

> > In v2.0 the `payNL.Transaction` methods will no longer be available. The transaction status endpoint will be kept for backwards compatibility.

Reviewer: ```suggestion
> In the upcoming v2.0 release the `payNL.Transaction` methods will no longer be available. The transaction status endpoint will be kept for backwards compatibility.
```
